matlab_simulation/05-estimation/kalman_filter/kf_example3.m

```diff
@@ -0,0 +1,105 @@
+% Kalman Filter Example 2: Multirate 2D Omnidirectional AUV example
```

Comment:
I got a weird error once while I was testing out this code. I didn't pay attention to it and was not able to recreate it. Not sure if this happened to you at all but I thought I would mention it.